snforge_std/src/cheatcodes.cairo

```diff
@@ -179,3 +185,39 @@ fn get_class_hash(contract_address: ContractAddress) -> ClassHash {
     let buf = cheatcode::<'get_class_hash'>(array![contract_address_felt].span());
     (*buf[0]).try_into().expect('Invalid class hash value')
 }
+
+trait L1Handler {
+    fn build(contract_address: ContractAddress, selector_name: felt252) -> L1HandlerFn;
+    fn invoke(self: L1HandlerFn, from_address: felt252, fee: u128, payload: Span::<felt252>);
+}
+
+impl L1HandlerImpl of L1Handler {
+    fn build(contract_address: ContractAddress, selector_name: felt252) -> L1HandlerFn {
+        L1HandlerFn {
+            contract_address,
+            selector_name,
+        }
+    }
+
+    fn invoke(self: L1HandlerFn, from_address: felt252, fee: u128, payload: Span::<felt252>) {
```

Comment:
The Fee here must never be 0, as the runner will fail due to the actual fee being higher than 0.
But if the user put at least 1, it's good as I used in the example.